ui/src/app/pages/data/data-set/new-dataset-modal.tsx

```diff
@@ -339,6 +351,42 @@ class NewDataSetModal extends React.Component<Props, State> {
                 </label>
               )}
           </React.Fragment>}
+          {this.props.displayMicroarrayOptions && this.state.kernelType === KernelTypeEnum.Python &&
```

Comment:
OK, forgot to flag one issue that I believe exists here: should ensure that the following doesn't 400 (leave it up to you the best method for doing so):

- check microarray
- switch to R
- export

Reply:
(right now, pretty sure the includeMicroarrayData will still be `true`, resulting in an export failure to R)

Reply:
good find. fixed!